org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/tracking/IssueTrackable.java

```diff
@@ -19,14 +19,19 @@
  */
 package org.sonarlint.eclipse.core.internal.tracking;
 
+import org.sonarlint.eclipse.core.internal.markers.TextRange;
 import org.sonarsource.sonarlint.core.client.api.common.analysis.Issue;
 
 public class IssueTrackable extends MutableTrackableImpl {
 
   private final Issue issue;
+  private final TextRange textRange;
+  private final Integer textRangeHash;
 
-  public IssueTrackable(Issue issue) {
+  public IssueTrackable(Issue issue, TextRange textRange, String textRangeContent) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "textRange". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1172)